R/get.R

```diff
@@ -629,6 +629,58 @@ get_oauth_credentials <- function(connect, user_session_token) {
   )
 }
 
+#' Perform an OAuth credential exchange to obtain a content-specific OAuth
+#' access token.
+#'
+#' @param connect A Connect R6 object.
+#' @param content_session_token The content session token. This token can only
+#' be obtained when the content is running on a Connect server. The token
+#' identifies the service account integration previously configured by the publisher
+#' on the Connect server.
+#'
+#' Read this value from the environment variable: `CONNECT_CONTENT_SESSION_TOKEN`
+#'
+#' @examples
+#' \dontrun{
+#' library(connectapi)
+#' library(plumber)
+#' client <- connect()
+#'
+#' #* @get /do
+#' function(req) {
+#'   content_session_token <- Sys.getenv("CONNECT_CONTENT_SESSION_TOKEN")
+#'   credentials <- get_oauth_content_credentials(client, content_session_token)
+#'
+#'   # ... do something with `credentials$access_token` ...
+#'
+#'   "done"
+#' }
+#' }
+#'
+#' @return The OAuth credential exchange response.
+#'
+#' @details
+#' Please see https://docs.posit.co/connect/user/oauth-integrations/#obtaining-a-content-oauth-access-token
```

Comment:
note: this doesn't exist yet but feels like a section we will add? Corresponds to the existing `#obtaining-a-viewer-oauth-access-token` section.

Reply:
here's the in-flight draft of the user guide changes: 
https://github.com/rstudio/connect/pull/29116

The parallel actually ended up being `#obtaining-a-service-account-oauth-access-token`